models/sla_policy/requester_wait_time/int_zendesk__requester_wait_time_business_hours.sql

```diff
@@ -120,18 +120,22 @@ with requester_wait_time_filtered_statuses as (
       weekly_period_requester_wait_time.week_number,
       weekly_period_requester_wait_time.ticket_week_start_time_minute,
       weekly_period_requester_wait_time.ticket_week_end_time_minute,
-      schedule.start_time_utc as schedule_start_time,
+      coalesce(schedule.start_time_utc, 0) as schedule_start_time,
       schedule.end_time_utc as schedule_end_time,
```

Comment:
This coalesce is necessary in place because the left join that follows can bring in null values. This is specific to `int_zendesk__requester_wait_time_business_hours` and `int_zendesk__agent_work_time_business_hours` but not the others since those go into ticket_metrics instead of sla_policies. I added in line comments.

Reply:
Gotcha! thanks for explaining